src/styles/details.scss

```diff
@@ -0,0 +1,165 @@
+.details {
+  &__title {
+    font-size: 32px;
+    line-height: 32px;
+    font-weight: 500;
+    text-align: center;
+    margin-top: 100px;
+    margin-bottom: 48px;
+  }
+
+  &__cards {
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
+    gap: 32px;
+
+    &-item {
+      display: flex;
+      flex-direction: column;
+
+      &-text {
+        font-size: 16px;
+        line-height: 22px;
+      }
+
+      &-images {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        gap: 20px;
+        margin-bottom: 8px;
+      }
+
+      &-img-first,
+      &-img-last {
+        width: 130px;
+        height: 130px;
+        object-fit: cover; /* Масштабувати зображення, зберігаючи пропорції */
+        object-position: center; /* Зосередження зображення по центру */
+      }
+
+      &-title {
+        font-size: 24px;
+        line-height: 33px;
+        font-weight: 700;
+        text-align: left;
+        margin-bottom: 10px;
+      }
+    }
+  }
+
+  &__button {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    &-item {
+      width: 278px;
+      height: 56px;
+      border-radius: 40px;
+      border: none;
+      border-color: $white-clr;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 27px;
+      color: $black-btn-clr;
+      margin-top: 40px;
+      margin-bottom: 80px;
+      transition: all 0.5s ease, transform 0.3s ease;
+
+
+
+      &:hover {
+        box-shadow: 0 0 10px 0 $white-shadow;
+      }
+    }
+  }
+
+  @media (min-width: 744px) {
+
+    &__title {
+      font-size: 48px;
+      line-height: 48px;
+      margin-bottom: 64px;
+    }
+
+    &__cards {
+      gap: 24px;
+
+      &-item {
+
+        &-images {
+          gap: 20px;
+          margin-bottom: 8px;
+        }
+
+        &-img-first {
+          width: 392px;
+          height: 187px;
+        }
+
+        &-img-last {
+          width: 184px;
+          height: 187px;
+          object-fit: cover;
+          object-position: center;
+        }
+      }
```

Comment:
Better way use 
&&__cards-item
&&__cards-item-images 

But in this case, better way create block for item. Then use item as block - item__images. 
-img-last and -img-first you need use as modifier (for markup - item__images--last, for scss --last --first)